### PR TITLE
fix/replace links

### PR DESCRIPTION
**Pull request by mairaw**
In PR #2780, @rpetrusha noticed we had quite a few API links still going to MSDN. I found a couple more not related to the feedback he was giving, so fixing some obvious ones on this PR + a few other broken/msdn links.

Related to #2580.

### CODE REVIEW

**Comment by rpetrusha** [comment]
An unspecified error occurred during the build; closing and reopening to restart build.

**Comment by rpetrusha** [comment]
@mairaw, the build has failed twice, so I suspect that one of the changes has caused the build break. I've looked through your changes, though, and I can't identify anything that might cause a build break.

**Review by BillWagner — APPROVED** [review]
I looked at all the diffs, and everything looks good. 

Let's :shipit: once we figure out why this build is failing.

File: docs/standard/framework-libraries.md [hunk 1]
```diff
@@ -26,33 +26,33 @@ The BCL provides the most foundational types and utility functionality and are t
...
+* <xref:System.Int16?displayProperty=fullName> ([short](../csharp/language-reference/keywords/short.md)) - A 16-bit signed integer type. The unsigned <xref:System.UInt16> also exists.
+* <xref:System.Int32?displayProperty=fullName> ([int](../csharp/language-reference/keywords/int.md)) - A 32-bit signed integer type. The unsigned [UInt32](../csharp/language-reference/keywords/uint.md) also exists.
+* <xref:System.Single?displayProperty=fullName> ([float](../csharp/language-reference/keywords/float.md)) - A 32-bit floating-point type.
+* <xref:System.Decimal?displayProperty=fullName> ([decimal](../csharp/language-reference/keywords/decimal.md)) - A 128-bit decimal type.
+* <xref:System.Byte?displayProperty=fullName> ([byte](../csharp/language-reference/keywords/byte.md)) - An unsigned 8-bit integer that represents a byte of memory.
+* <xref:System.Boolean?displayProperty=fullName> ([bool](../csharp/language-reference/keywords/bool.md)) - A boolean type that represents ‘true’ or ‘false’.
+* <xref:?displayProperty=fullName> ([char](../csharp/language-reference/keywords/char.md)) - A 16-bit numeric type that represents a Unicode character.
```
Comment (vicancy) [inline docs/standard/framework-libraries.md]: empty uid, should be `<xref:char?displayProperty=fullName>`

Comment (mairaw) [inline docs/standard/framework-libraries.md]: Thanks @vicancy!

**Pull request merged**